apps/site/src/views/Resources/components/ResourceCard/ResourceCard.tsx

```diff
@@ -0,0 +1,63 @@
+import Image from "next/image";
+
+import openNewWindow from "@/assets/icons/open-new-window.svg";
+import styles from "./ResourceCard.module.scss";
+
+type Tag = {
+	text: string;
+	link: string;
+};
+
+interface ResourceCardProps {
+	title: string;
+	description: JSX.Element;
+	stickerSrc?: string;
+	links: Tag[];
+	stickyNoteColor: string;
+}
+
+export default function ResourceCard({
+	title,
+	description,
+	stickerSrc,
+	links,
+	stickyNoteColor,
+}: ResourceCardProps) {
+	return (
+		<div
+			className={styles.group}
+			style={{
+				backgroundColor: `${stickyNoteColor}`,
+			}}
+		>
+			<div className={styles.container + " text-center px-3"}>
+				<div className={styles.tape}></div>
+				{stickerSrc ? (
+					<img src={stickerSrc} alt="Resource logo" width="100" />
+				) : null}
```

Comment:
Suggestion [non-blocking]: falsy values are not rendered, so the conditional could be short-circuited
```jsx
{stickerSrc && <img .../>}
```